proof/refine/AARCH64/IpcCancel_R.thy

```diff
@@ -539,9 +515,12 @@ lemma (in delete_one) cancelIPC_ReplyCap_corres:
                in corres_underlying_split)
      apply (rule corres_guard_imp)
        apply (rule threadset_corresT)
-          apply (simp add: tcb_relation_def fault_rel_optionation_def)
-         apply (simp add: tcb_cap_cases_def)
-        apply (simp add: tcb_cte_cases_def cteSizeBits_def)
+             apply (simp add: tcb_relation_def fault_rel_optionation_def)
+            apply (simp add: tcb_cap_cases_def)
+           apply (simp add: tcb_cte_cases_def cteSizeBits_def)
+          apply simp
+         apply simp
+        apply simp
```

Comment:
this can likely be shortened, at minimum all subgoals generated from the `rule` get a simp, so a `; simp add: ...` can get rid of at least 3